packages/smooth_app/lib/pages/product/product_image_gallery_view.dart

```diff
@@ -136,31 +136,27 @@ class _ProductImageGalleryViewState extends State<ProductImageGalleryView> {
                               if (!mounted) {
                                 return;
                               }
-                              final bool isUploaded =
-                                  await uploadCapturedPicture(
+                              await uploadCapturedPicture(
                                 context,
                                 barcode: widget.barcode!,
                                 imageField: _productImageDataCurrent.imageField,
                                 imageUri: croppedImageFile.uri,
                               );
 
-                              if (isUploaded) {
-                                _isRefreshed = true;
-                                if (!mounted) {
-                                  return;
-                                }
-                                final AppLocalizations appLocalizations =
-                                    AppLocalizations.of(context);
-                                final String message = getImageUploadedMessage(
-                                    _productImageDataCurrent.imageField,
-                                    appLocalizations);
-                                ScaffoldMessenger.of(context).showSnackBar(
-                                  SnackBar(
-                                    content: Text(message),
-                                    duration: const Duration(seconds: 3),
-                                  ),
-                                );
+                              _isRefreshed = true;
```

Comment:
Depends on what you mean with `_isRefreshed`.
In the previous version it could mean the picture was uploaded.

Reply:
I guess i have to remove this _isRefreshed here  , after scheduling the task not sure if it will run just instantly here